process/accounts/accountsProcessor.go

```diff
@@ -281,37 +259,14 @@ func (ap *accountsProcessor) PutTokenMedataDataInTokens(tokensData []*data.Token
 	}
 }
 
-func (ap *accountsProcessor) loadMetadataFromSystemAccount(tokenKey []byte) (*esdt.MetaData, error) {
-	systemAccount, err := ap.accountsDB.LoadAccount(vmcommon.SystemAccountAddress)
-	if err != nil {
-		return nil, err
-	}
-
-	userAccount, ok := systemAccount.(coreData.UserAccountHandler)
-	if !ok {
-		return nil, indexer.ErrCannotCastAccountHandlerToUserAccount
-	}
-
-	marshaledData, err := userAccount.RetrieveValueFromDataTrieTracker(tokenKey)
-	if err != nil {
-		return nil, err
-	}
-
-	esdtData := &esdt.ESDigitalToken{}
-	err = ap.internalMarshalizer.Unmarshal(esdtData, marshaledData)
-	if err != nil {
-		return nil, err
-	}
-
-	return esdtData.TokenMetaData, nil
-}
-
-func computeTokenKey(token string, nonce uint64) []byte {
-	tokenKey := []byte(core.ElrondProtectedKeyPrefix + core.ESDTKeyIdentifier + token)
-	if nonce > 0 {
-		nonceBig := big.NewInt(0).SetUint64(nonce)
-		tokenKey = append(tokenKey, nonceBig.Bytes()...)
+func (ap *accountsProcessor) loadMetadataForToken(tokenData *data.TokenInfo, coreAlteredAccounts map[string]*coreIndexerData.AlteredAccount) (*esdt.MetaData, error) {
```

Comment:
split function definition in multiple lines

Reply:
done